skribenten-backend/src/test/kotlin/no/nav/pensjon/brev/skribenten/auth/AuthorizeAnsattSakTilgangTest.kt

```diff
@@ -244,7 +250,7 @@ class AuthorizeAnsattSakTilgangTest {
         coEvery { navansattService.hentNavAnsattEnhetListe(any(), any()) } returns ServiceResult.Error("Ansatt finnes ikke", HttpStatusCode.NotFound)
 
         val response = client.get("/sak/${testSak.saksId}")
-        assertEquals(HttpStatusCode.InternalServerError, response.status)
+        assertEquals(HttpStatusCode.NotFound, response.status)
```

Comment:
Navnet på testen sier at det forventes Internal Server Error. Og det synes jeg gir mening. Om vi returnerer Not Found her så indikerer vi at saken ikke finnes, mens det som egentlig er tilfeller er at vi mangler informasjon om ansatt.

Reply:
jeg er enig, men det blir potensielt litt grisete kode å spesialhåndtere en spesifik feilkode langt inni koden og forstå at det er for å fikse et aggregert svar på en tjeneste

Reply:
Jeg er enig men det blir potensielt litt grisete å se på spesialhåndtering av en konkret feilkode etter await langt inni koden der